src/hotspot/share/services/mallocTracker.hpp

```diff
@@ -152,8 +152,13 @@ class MallocMemorySnapshot : public ResourceObj {
   MallocMemory      _malloc[mt_number_of_types];
   MemoryCounter     _all_mallocs;
 
+  // Used only for ThreadStackTracker::track_as_vm() == true to report thread
+  // count diff during "jcmd <pid> VM.native_memory summary.diff" correctly.
+  int               _thread_count;
```

Comment:
`_thread_count` should be of `size_t` type

Reply:
Fixed. Thanks for catching this